nixos/modules/services/security/tor.nix

```diff
@@ -146,7 +146,7 @@ let
     ]))];
     description = lib.mdDoc (descriptionGeneric optionName);
   };
-  optionBandwith = optionName: mkOption {
+  optionBandwidth = optionName: mkOption {
```

Comment:
That one probably needs an alias for backwards compatibility.

Reply:
I think this is a let binding that shouldn't break any backwards compatibility